reconcile/openshift_groups.py

```diff
@@ -111,6 +115,14 @@ def fetch_desired_state(oc_map):
     desired_state = []
 
     for r in roles:
+        gql.get_api
+        checkExpirationDate = openshift_resource.checkExpirationDate(r)
+        if checkExpirationDate is True:
+            continue
+        elif checkExpirationDate is False:
```

Comment:
PEP8 requires snake_case instead of camelCase, so please `check_expiration_date()` and `check_expiration_date`.